zppy/templates/coupled_global.py

```diff
@@ -853,7 +1132,49 @@ def run_by_region(parameters):
         else:
             raise RuntimeError(f"Invalid rgn={rgn}")
         run(parameters, rgn)
+    figstr = parameters[3]
+    # plots_original = param_get_list(parameters[8])
+    # if parameters[9].lower() == "false":
+    #     atmosphere_only = False
+    # else:
+    #     atmosphere_only = True
+    # plots_atm = param_get_list(parameters[10])
+    # plots_ice = param_get_list(parameters[11])
+    plots_lnd = param_get_list(parameters[12])
+    # plots_ocn = param_get_list(parameters[13])
+    nrows = int(parameters[14])
+    ncols = int(parameters[15])
+    plots_per_page = nrows * ncols
+    if plots_per_page == 1:
+        # In this case, we don't want the summary PDF.
+        # Rather, we want to construct a viewer similar to E3SM Diags.
+        # TODO: make viewer home page to point to multiple viewers
+        url = create_viewer(figstr, regions, "lnd", plots_lnd)
```

Comment:
TODO: Right now, the viewer only exists for land variables. We'll need to create a viewer for each component (e.g., atm, ice, ocn). 

- Design decision: In the absence of corresponding variable definition csv files for those components, I can just put all the component's variables under one group.
- To make a home page of multiple viewers, we'll have to more closely emulate the E3SM Diags home page (e.g., https://github.com/E3SM-Project/e3sm_diags/blob/main/e3sm_diags/viewer/main.py, https://github.com/E3SM-Project/e3sm_diags/blob/main/e3sm_diags/viewer/index_template.html)

(Again, a note on design and tech debt: this is very much outside the scope of zppy as a post-processing workflow coordinator. At this point, Global Time Series is for all intents-and-purposes a plotting package fully contained within `zppy` due to legacy reasons).